test-manager.go

```diff
@@ -196,13 +219,13 @@ func (m *TestManager) GetTestDir() string {
 
 // Reverts the services to the baseline snapshot
 func (m *TestManager) RevertToBaseline() error {
-	err := m.revertToSnapshot(m.baselineSnapshotID)
+	err := m.RevertToSnapshot(m.baselineSnapshotID)
 	if err != nil {
 		return fmt.Errorf("error reverting to baseline snapshot: %w", err)
 	}
 
 	// Regenerate the baseline snapshot since Hardhat can't revert to it multiple times
-	baselineSnapshotID, err := m.takeSnapshot(Service_All)
+	baselineSnapshotID, err := m.CreateSnapshot()
```

Comment:
This is obsolete now, since `RevertToSnapshot()` already regenerates the hardhat snapshot. You can just call `RevertToSnapshot(m.baselineSnapshotID)` above and return straight away.

Reply:
done